puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/build/rubygem-katello-nightly-release.yaml

```diff
@@ -4,6 +4,20 @@
     sandbox: false
     triggers:
       - github
+    parameters:
+      - string:
+          name: OWNER_REPO
+          default: Katello/katello
+      - string:
+          name: BRANCH
+          default: master
+      - string:
+          name: GEMSPEC
+          default: katello.gemspec
+      - string:
+          name: PACKAGE_NAME
+          default: rubygem-katello
```

Comment:
The weird part about parameters is, you can change them per job invocation which seems odd for most of these to do.